### DISCUSSION TITLE
Regridding forecast data

### DISCUSSION THREAD

**Discussion by mjohnson52** [dtcenter/METplus#2231] [Configuration] [answered]
Hello,

I'm trying to regrid high-resolution data to a coarser grid. Many papers that utilize "surrogate severe reports" use an 80 km grid, and flag the coarser grid box if any high-resolution grid points within the 80 km grid box exceed a certain threshold. Is there a way to regrid in a similar manner in MET (i.e., take the maximum value of high resolution grid points within the coarser grid box)? I can see "MAX" can be selected as the regrid option with a provided width, but I'm unsure of how these are practically applied when regridding. 

Thanks!
Marcus

**Comment by TaraJensen** [comment, marked as answer]
Marcus,

There are two use cases in the METplus documentation and repository.  Please see:

- https://metplus.readthedocs.io/en/latest/generated/model_applications/short_range/GridStat_fcstHRRR_obsPracPerfect_SurrogateSevere.html
- https://metplus.readthedocs.io/en/latest/generated/model_applications/short_range/GridStat_fcstHRRR_obsPracPerfect_SurrogateSevereProb.html?highlight=ShortRangeAppUseCase

Let us know if you need additional help.

**Comment by mjohnson52** [comment]
Hi Tara,

Thank you for the above resources for SSR and practically perfect verification!

Marcus
